djangocms_frontend/contrib/navigation/templates/djangocms_frontend/bootstrap5/navigation/default/dropdown.html

```diff
@@ -3,8 +3,8 @@
         <li class="nav-item dropdown">
             <a class="nav-link dropdown-toggle" role="button"
                data-bs-toggle="dropdown" href="#"
-               id="menu-{{instance.id}}">{{ instance.title }}</a>
-                <div class="dropdown-menu" aria-labelledby="menu-{{instance.id}}">
+               id="menu-{{ instance.id|safe }}">{{ instance.title }}</a>
```

Comment:
I've never done this. Curious as to why it does.

Reply:
I use `instance.id|safe` in stead of `instance.id|unlocalize` since I do not have to import `l10n` into each and every template (`safe` is build in). It marks the content as safe and thereby prevents any formatting or escaping. This way the number gets included into the template without thousand separators.  It'll be `menu-1001` and not `menu-1 001` or something.